src/lib.rs

```diff
@@ -117,7 +117,7 @@
 //!
 //! ```no_run
 //! use std::io::{stdout, Write};
-//! use crossterm::{ExecutableCommand, QueueableCommand, Color, PrintStyledFont, Colorize, Clear, ClearType, Goto, Result};
+//! use crossterm::{ExecutableCommand, QueueableCommand, Color, PrintStyledFont, Colorize, Clear, ClearType, cursor::MoveTo, Result};
```

Comment:
Long line. Horizontal scrollbar in the documentation, because it shows just:

```
use crossterm::{ExecutableCommand, QueueableCommand, Color, PrintStyledFont, Colorize, Clear, Cl
```